values.yaml

```diff
@@ -24,6 +24,8 @@ identity:
       # @ignore
       pullPolicy: IfNotPresent
   krbKeysOperator:
+    # -- A comma-separated list of namespaces that the KerberosKey Operator should watch for KerberosKey resources in. Defaults to the release namespace if not specified
+    namespaces: {}
```

Comment:
It's a little confusing giving an object here when any non-empty value needs to be a string. I would use `""` or `null` instead.